server/proxy/filters/filters_api.h

```diff
@@ -22,6 +22,20 @@
 #ifndef FREERDP_SERVER_PROXY_FILTERS_API_H
 #define FREERDP_SERVER_PROXY_FILTERS_API_H
 
+#if defined _WIN32 || defined __CYGWIN__
```

Comment:
IIRC we have something equivalent somewhere else, can you check that  ?

Reply:
We have FREERDP_API/WINPR_API
Maybe i can #define PROXY_API FREERDP_API

Reply:
sounds nice

Reply:
ok please test again